src/screens/HomeScreenV2/sections/MainBannerSection.tsx

```
@@ -220,7 +227,7 @@ export default function MainBannerSection({
                 style={{
                   left: item.slot * ITEM_SLOT_WIDTH + BANNER_HORIZONTAL_PADDING,
                 }}>
-                <MainBanner banner={item.banner} index={item.ringIndex} />
+                <MainBanner banner={item.banner} index={item.ringIndex} trackView={item.slot === renderCenter} />
```

ESLint

[error] Replace `·banner={item.banner}·index={item.ringIndex}·trackView={item.slot·===·renderCenter}` with `⏎··················banner={item.banner}⏎··················index={item.ringIndex}⏎··················trackView={item.slot·===·renderCenter}⏎···············`

(prettier/prettier)